### PR TITLE
Engine: Fix inputs in Lefty mode

### LINKED ISSUE #1: Inputs don't respect the Lefty mode
Best observed on Air Arkanoid, but any other games will be affected.

1. On the Flipper, set the Hand Orientation option to Lefty.
2. Boot Air Arkanoid.
3. Observe that navigating down to "Settings" requires pressing dpad down.
4. Start the game.
5. Observe that moving the paddle left requires pressing dpad right, and vice versa.

Inputs received via an input subscription don't seem to be inverted, usually it's done by the viewport - but since the Flipper Zero Game Engine uses Direct Draw, this inversion isn't taking place.

### PR DESCRIPTION
**Pull request by CookiePLMonster**
Fixes #1.

### CODE REVIEW

File: game_engine.c [hunk 1]
```diff
@@ -70,11 +71,22 @@ static const GameKey keys[] = {
...
+static const GameKey keys_left_hand[] = {
+    [InputKeyUp] = GameKeyDown,
+    [InputKeyDown] = GameKeyUp,
+    [InputKeyRight] = GameKeyLeft,
+    [InputKeyLeft] = GameKeyRight,
+    [InputKeyOk] = GameKeyOk,
+    [InputKeyBack] = GameKeyBack,
+};
+
+static const size_t keys_count = sizeof(keys_right_hand) / sizeof(keys_right_hand[0]);
```
Comment (DrZlo13) [inline game_engine.c:91]: please add a static assertion to check that the sizes of the_left_hand keys and the right_hand_keys are equal

**Review by DrZlo13 — CHANGES_REQUESTED** [review]

File: game_engine.c (hunk 1 above)
Comment (CookiePLMonster) [reply game_engine.c:91]: Done!

**Comment by DrZlo13** [comment]
@CookiePLMonster I did the lefty flag reading only once and added lefty mode support to the IMU. Check it out and approve if this is the correct way.

**Comment by CookiePLMonster** [comment]
Excellent! I like the idea of reading the flag once, too. Viewports did not do that but it makes more sense as they are part of firmware, while this isn't so the user can't reasonably change this flag in-app.

**Pull request merged**